tools/jdbc/src/jni/duckdb_java.cpp

```diff
@@ -310,13 +312,22 @@ static const char *const JDBC_STREAM_RESULTS = "jdbc_stream_results";
 jobject _duckdb_jdbc_startup(JNIEnv *env, jclass, jbyteArray database_j, jboolean read_only, jobject props) {
 	auto database = byte_array_to_string(env, database_j);
 	DBConfig config;
+	config.options.duckdb_api = "jni";
```

Comment:
Why not call this Java? You also seem to be overriding it in the Java layer, which is a bit confusing

Reply:
Yep, I'll rename to "java". This override is here just in case someone uses Java without JDBC -- making the distinction  between "java" and "jdbc" would make this obscure case easier to troubleshoot.

Reply:
I don't know if that would be possible? The Java API *is* the the JDBC API